client/boot/common.js

```diff
@@ -264,10 +296,22 @@ function setupErrorLogger( reduxStore ) {
 		errorLogger.saveExtraData( { lastTracksEvent } )
 	);
 
+	let prevPath;
 	page( '*', function ( context, next ) {
-		errorLogger.saveNewPath(
-			context.canonicalPath.replace( getSiteFragment( context.canonicalPath ), ':siteId' )
+		const path = context.canonicalPath.replace(
+			getSiteFragment( context.canonicalPath ),
+			':siteId'
 		);
+		Sentry.addBreadcrumb( {
+			type: 'navigation',
+			data: {
+				from: prevPath ?? path,
+				to: path,
```

Comment:
While the default navigation handlers do include the path, it also includes the site ID. Following how it worked previously, it makes it easier to work with when we normalize the URL -- so instead of `/home/example.wordpress.com`, it should show `/home/:siteId`, just like logstash.